models/wrf_hydro/noah_hydro_mod.f90

```diff
@@ -1028,23 +1032,24 @@ subroutine get_routelink_constants(filename)
 ! length: Length (Stream length (m))
 !     to: "To Link ID (PlusFlow table TOCOMID for every FROMCOMID)"
 
-call nc_get_variable(ncid,'lon',   linkLong,  routine)
-call nc_get_variable(ncid,'lat',   linkLat,   routine)
-call nc_get_variable(ncid,'alt',   linkAlt,   routine)
-call nc_get_variable(ncid,'n',     roughness, routine)
-call nc_get_variable(ncid,'link',  linkID,    routine)
-call nc_get_variable(ncid,'Length',length,    routine)
-call nc_get_variable(ncid,'to',    to,        routine)
+call nc_get_variable(ncid,'lon',              linkLong,  routine)
+call nc_get_variable(ncid,'lat',              linkLat,   routine)
+call nc_get_variable(ncid,'alt',              linkAlt,   routine)
+!call nc_get_variable(ncid,'n',                roughness, routine)
+!call nc_get_variable(ncid,'link',             linkID,    routine)
+call nc_get_variable(ncid,'Length',           length,    routine)
+call nc_get_variable(ncid,'to',               to,        routine)
 call nc_get_variable(ncid,'bucket_comid_mask',BucketMask,routine)
 
 ! no snappy accessor routine for character arrays
 ! call nc_get_variable(ncid,'gages', gageID,    routine)
-io = nf90_inq_varid(ncid,'gages', VarID)
-call nc_check(io, routine, 'inq_varid', 'gages', filename)
-io = nf90_get_var(ncid, VarID, gageID)
-call nc_check(io, routine, 'get_var', 'gages', filename)
 
-call nc_close_file(ncid, routine)
+!io = nf90_inq_varid(ncid,'gages', VarID)
+!call nc_check(io, routine, 'inq_varid', 'gages', filename)
+!io = nf90_get_var(ncid, VarID, gageID)
+!call nc_check(io, routine, 'get_var', 'gages', filename)
+
+!call nc_close_file(ncid, routine)
```

Comment:
There are quite a few commented out lines of code. It would be better to remove the code if it is not being used.

Reply:
Done